src/Arcus.Messaging.Tests.Workers.ServiceBus.Topic/Dockerfile

```
@@ -1,7 +1,7 @@
-FROM mcr.microsoft.com/dotnet/core/aspnet:3.0-alpine3.9 AS base
+FROM mcr.microsoft.com/dotnet/core/sdk:3.1.202-alpine3.10 AS base
```

You need to use asp as base
```suggestion
FROM mcr.microsoft.com/dotnet/core/aspnet:3.1-alpine3.11 AS base
```